crates/turbopack-core/src/chunk/availability_info.rs

```diff
@@ -1,19 +1,56 @@
+use std::ops::{Add, AddAssign};
+
 use turbo_tasks::Vc;
 
 use super::available_modules::AvailableAssets;
 use crate::module::Module;
 
+#[turbo_tasks::value(shared)]
+#[derive(Copy, Clone, Debug)]
+pub enum AvailabilityInfoNeeds {
```

Comment:
Wouldn't this be better off as a struct with 2 boolean fields?

And then implement `BitOr` instead of `Add`

Reply:
yeah that makes sense...